dbt/adapters/snowflake/impl.py

```diff
@@ -419,3 +421,18 @@ def valid_incremental_strategies(self):
     def debug_query(self):
         """Override for DebugTask method"""
         self.execute("select 1 as id")
+
+    @classmethod
+    def _get_adapter_specific_run_info(cls, config: RelationConfig) -> Dict[str, Any]:
```

Comment:
Do we want `None`, `""`, or to omit `table_format` entirely when it doesn't exist? Does this output align with the jsonschema? (it may, just validating)

Reply:
It does align with the Json schema now that I've added the null case to the upstream. Thanks for pointing this out